packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -72,6 +81,12 @@ mkdir -p %{buildroot}%{_bindir}
 cp -pa .%{_bindir}/foreman-node %{buildroot}%{_bindir}/foreman-node
 cp -pa .%{gem_instdir}/sbin/upload-salt-reports %{buildroot}%{_sbindir}/upload-salt-reports
 install -Dp -m0644 .%{gem_instdir}/cron/smart_proxy_salt %{buildroot}%{_sysconfdir}/cron.d/%{gem_name}
+mkdir -p %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}
+cat <<EOF > %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}/smart_proxy_salt_core.rb
+gem 'smart_proxy_salt_core'
+EOF
+mkdir -p %{buildroot}%{foreman_proxy_dir}
+cp -pa .%{gem_instdir}/settings.d/salt.saltfile.example %{buildroot}%{foreman_proxy_dir}/Saltfile
```

Comment:
In https://github.com/theforeman/smart_proxy_salt/blob/d97f20da1c78ccc7527365a0177cb1cd2738b298/settings.d/salt.yml.example#L12 the location of the file is '/etc/foreman-proxy/settings.d/salt.saltfile'